features/step_definitions/books.rb

```diff
@@ -0,0 +1,68 @@
+Given('I have access to the book list') do
+  wait_for_element_to_be_ready(:id, 'book_list')
+  wait(1)
```

Comment:
Why do you need this?

Reply:
you can implement this action in another step

Reply:
if i don't use the wait, the books don't have enough time to load so i can't create the array with `find_elements` (the array is empty). I know it's not the right way but i couldn't find an explicit wait that fits